app/config/config.go

```diff
@@ -15,8 +15,9 @@ const DEFAULT_CONFIG string = "/etc/susetelemetry/server.cfg"
 
 // API server config
 type APIConfig struct {
-	Host string `yaml:"host"`
-	Port int    `yaml:"port"`
+	Host           string `yaml:"host"`
+	Port           int    `yaml:"port"`
+	MaxParallelReq int    `yaml:"max_parallel_requests"`
```

Comment:
We may want to add min, max and default constants for this setting, and potentially add a validity checking routine of some sort for the APIConfig struct that validates the configured value falls within the permitted range.

Reply:
We probably also want it to default to the default constant value if not specified, and I would make that value 50 rather than 80, with 80 as the max, and maybe 3 or 5 as the min...